spec/src/main/asciidoc/jsonb.adoc

```diff
@@ -1,3 +1,769 @@
+:sectnums:
+= Jakarta JSON Binding Specification, Version 1.0
+
+Copyright (c) 2019 Eclipse Foundation 
+
 == Introduction
 
-Jakarta JSON Binding defines a binding framework for converting Java(R) objects to and from JSON documents.
+This specification defines binding API between Java objects and https://tools.ietf.org/html/rfc7159[JSON] documents. Readers are assumed to be familiar with JSON; for more information about JSON, see:
+
+* https://www.ics.uci.edu/~fielding/pubs/dissertation/fielding_dissertation.pdf[Architectural Styles and the Design of Network-based Software Architectures]
+* http://rest.blueoxen.net/cgi-bin/wiki.pl[The REST Wiki]
+* http://en.wikipedia.org/wiki/JSON[JSON on Wikipedia]
+
+=== Status
+
+A list of open issues can be found at:
+
+____
+https://github.com/eclipse-ee4j/jsonb-api/issues
+____
+
+The corresponding source code can be found online at:
+
+____
+https://github.com/eclipse-ee4j/jsonb-api
+____
+
+Compatible implementations can be found at:
+
+* https://github.com/eclipse-ee4j/yasson[Eclipse Yasson]
+* https://github.com/apache/johnzon/tree/master/johnzon-jsonb[Apache Johnzon]
```

Comment:
It's certainly convenient to have these links to the implementations.  However, it also seems out of place to be identifying the list of compatible implementations within the specification document itself.  In normal cases, the specification would need to be written first, and then implementations could become compatible with it.  Is there a better place we could include this information?

Reply:
Yea it does seem a bit backwards, I'll update the user-guide to have this information instead